include/revng/Support/CommandLine.h

```diff
@@ -9,6 +9,10 @@
 #include "llvm/Support/CommandLine.h"
 
 extern llvm::cl::OptionCategory MainCategory;
+
+// This is the path of the input binary. CLI tools should populate this on
+// every invocations. Non-CLI users of this variable need to account for it
+// being possibly empty.
```

Comment:
1. Use Doxygen
2. > Non-CLI users of this variable need to account for it being possibly empty

   Users should not be aware whether they are being used by the CLI or not, they should just assume this might be empty.

Reply:
1. Done
2. Reworded